### PR TITLE
Add benefit hubs and rearrange ending components for LC templates

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
**Issue:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/14330

This PR adds benefit hubs and rearranges the ending LC components (bread layers) according to the updated [design spec](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1588167553516/2773c854e8ff3a725a5e3ee03272b78e8519f78b).

This PR now also refactors `Needs more help` to derive it's information from new key-value pairs from the CMS:

image

image

image

## Testing done
N/A

## Screenshots
localhost_3001_preview_nodeId=6898

## Acceptance criteria
- [x] Add benefit hubs
- [x] Update ordering of ending components for each LC page

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by ncksllvn — APPROVED** [review]
LGTM!

**Review by jbalboni — APPROVED** [review]

File: src/site/includes/how-do-you-rate.drupal.liquid [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% if false %}
```
Comment (kelsonic) [inline src/site/includes/how-do-you-rate.drupal.liquid:1]: This isn't used currently

Comment (ncksllvn) [reply src/site/includes/how-do-you-rate.drupal.liquid:1]: can also `{% comment %}`

**Review by ncksllvn — APPROVED** [review]
LGTM, great job!

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**